RELEASE-NOTES.md

```diff
@@ -38,11 +38,12 @@ Other breaking changes:
 * Removed previously deprecated `Property::newEmpty`, use `Property::newFromType` or `new Property()` instead
 * Renamed `StatementList::getWithPropertyId` to `StatementList::getByPropertyId`
 * Renamed `StatementList::getWithRank` to `StatementList::getByRank`
-* `Reference` and `ReferenceList`s no longer can not be instantiated with `null`
-* Added `setId` method to `EntityDocument`
+* `Entity::setLabel` and `Entity::setDescription` no longer return anything
+* `Reference` and `ReferenceList`s no longer can be instantiated with `null`
 
 #### Additions
 
+* Added `EntityDocument::setId`
```

Comment:
How is a method added to an interface a breaking change? I can't think of a way how does this could break existing code.